sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreIntegrationSuite.scala

```diff
@@ -21,15 +21,168 @@ import java.io.File
 
 import scala.jdk.CollectionConverters.SetHasAsScala
 
+import org.apache.hadoop.conf.Configuration
 import org.scalatest.time.{Minute, Span}
 
+import org.apache.spark.sql.catalyst.expressions.UnsafeRow
 import org.apache.spark.sql.execution.streaming.{MemoryStream, StreamingQueryWrapper}
 import org.apache.spark.sql.functions.count
 import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.streaming._
 import org.apache.spark.sql.streaming.OutputMode.Update
+import org.apache.spark.sql.types.StructType
 import org.apache.spark.util.Utils
 
+object TestStateStoreWrapper {
```

Comment:
Probably will want a better name here.

```suggestion
object CheckpointInfoCollectingStateStore {
```

?